core/src/audio_key.rs

```
@@ -32,11 +32,11 @@ impl AudioKeyManager {
                 0xd => {
                     let mut key = [0u8; 16];
                     key.copy_from_slice(data.as_ref());
-                    sender.complete(Ok(AudioKey(key)));
+                    drop(sender.send(Ok(AudioKey(key))));
```

`let _ =` is a more idiomatic way to ignore errors